component/card/card.css

```diff
@@ -0,0 +1,144 @@
+
+  .card-hor {
+    display: flex;
+    width: 100%;
+    max-width: 700px;
+    border-radius: 0.5rem;
+    margin-bottom: 2rem;
+    background-color: var(--light-color);
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  }
+  .card-hor-image-ctn {
+    flex: 0.7;
+    padding: 3%;
+  }
+  .card-hor-image-ctn img {
+    border-radius: 1rem;
+  }
+  
+  .card-hor-text-ctn {
+    flex: 1;
+    padding: 1rem;
+    box-sizing: border-box;
+  }
+  .card-hor-image-ctn {
+    max-width: 170px;
+    height: auto;
+  }
```

Comment:
You have created (card-hor-image-ctn) class twice.
See on line 11

Reply:
made changes in it !!